packages/styles/scss/patterns/blocks/content-block-simple/README.md

```diff
@@ -0,0 +1,38 @@
+### SCSS
+
+#### Usage
+
+Import the package css into the top of your main CSS file.
+
+```css
+@import '@carbon/ibmdotcom-styles/scss/patterns/blocks/content-block-simple/index';
+```
+
+#### Modifiers
+
+Use these modifiers with `.bx--content-block-simple` class.
+
+| Class                                    | Description                                            |
+| ---------------------------------------- | ------------------------------------------------------ |
+| `.bx--content-block-simple--icon-link`   | The class name for simple long form with icon link     |
+| `.bx--content-block-simple--card`        | The class name for simple long form with card link     |
+| `.bx--content-block-simple--with-border` | The class name for simple long form with border bottom |
```

Comment:
I'm not seeing these anywhere else, am I overlooking something?

Reply:
No, you're right. I didn't realize we even had READMEs for css...